miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditRunController.java

```diff
@@ -312,4 +320,85 @@ public String processSubmit(@ModelAttribute("run") Run run, ModelMap model, Sess
       throw ex;
     }
   }
+
+  @Value("${miso.run.report.links:}")
+  private String runReportLinksConfigLine;
+
+  private Map<String, String> getRunReportServicesUrls(Run run) {
+    Map<String, String> runReportLinks = new TreeMap<>();
+    if (isStringBlankOrNull(runReportLinksConfigLine) || run.getId() == Run.UNSAVED_ID) {
+      return runReportLinks;
+    }
+
+    // config format: "<PlatformType, PlatformType>:<link text>:<use HTTPS>:<domain>:<port>:<page address>:<run field>"
+    // e.g. "Illumina:Run Report:true:local.domain:8080:/runReport?id=~:id"
+    // `~` is used in the <page address> as a placeholder for where the <run field> goes.
+    // multiple run report links may be comma-separated
+    String[] configStrings = runReportLinksConfigLine.split(",");
+    for (int i = 0; i < configStrings.length; i++) {
+      String[] configParts = configStrings[i].split(":");
+      if (configParts.length != 7) {
+        log.info(String.format("Invalid run report links configuration: %s", configStrings[i]));
+        continue;
+      }
+
+      Set<String> platformTypeStrings = Sets.newHashSet(configParts[0].trim().split(","));
+      Set<PlatformType> platformTypes = platformTypeStrings.stream().map(pt -> PlatformType.get(pt.trim())).collect(Collectors.toSet());
+      String linkText = configParts[1].trim();
+      boolean useHttps = Boolean.valueOf(configParts[2].trim());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use "Boolean.parseBoolean" for this string-to-boolean conversion. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2130)